tests/stdlib/tmath.nim

```diff
@@ -338,8 +338,11 @@ template main =
     doAssert copySign(10.0, 0.0) == 10.0
     doAssert copySign(-1.0, NaN) == 1.0
     doAssert copySign(10.0, NaN) == 10.0
-    doAssert copySign(-1.0, -NaN) == 1.0
-    doAssert copySign(10.0, -NaN) == 10.0
+
+    when nimvm: discard
+    else:
+      doAssert copySign(-1.0, -NaN) == 1.0
```

Comment:
this is a bug, we should track it

Reply:
EDIT: caused by https://github.com/timotheecour/Nim/issues/499

Reply:
Ok

Reply:
Why disable it? It works for me.

Reply:
Why disable it? It works for me.

Reply:
maybe it's osx specific? what do you get for https://github.com/timotheecour/Nim/issues/499 ?

Reply:
I don't know. It seems that It works with `koch temp c -r` but doesn't work with `nim c -r --lib:lib` and Nim CI>

Reply:
> maybe it's osx specific? what do you get for #499 ?

#499 is not OS specific, I got the same result in windows